content/blog/2020-07-22-july-20-community-gems.md

```diff
@@ -0,0 +1,183 @@
+
+If you're installing DVC from binary-such as the `dvc.exe`
+[downloadable on the DVC homepage](https://dvc.org/)- all the standard
+dependencies are included. You shouldn't need to use `pip` to install extra
+packages (like `boto` for S3 storage).
+
+### [Q: Is there a way to setup my DVC remote so I can manually download files from it without going through DVC?](https://discordapp.com/channels/485586884165107732/563406153334128681/717458695709130764)
+
+When DVC adds a file to a remote repository (such as an S3 bucket, or an SSH
+file server), there's only one change happening: DVC calculates an md5 for the
+file and renames it with that md5. In technical terms, it's storing files in a
+"content-addressable way". That means if you know the hash of a file, you can
+locate it in your DVC remote and manually download it.
+
+To find the hash for a given file, say `data.csv`, you can look in the
+corresponding DVC file:
+
+```dvc
+$ cat data.csv.dvc
+```
+
+Another approach is using a built-in DVC function:
+
+```dvc
+$ dvc get --show-url . data.csv
+```
+
+You can read more about `dvc get --show-url` in
+[our docs](https://dvc.org/doc/command-reference/get#options). Note that this
+functinality is also part of our Python API, so you can locate the path to a
+file in your remote within a Python environment.
+[Check out our API docs!](https://dvc.org/doc/api-reference/get_url)
+
+### [Q: By default, each DVC project has its own cache in the project repository. To save space, I'm thinking about locally creating a single cache folder and letting multiple project repositories point there. Will this work?](https://discordapp.com/channels/485586884165107732/563406153334128681/736164141701791815)
+
+Yes, we hear from many users who have created a shared cache. Because of the way
+DVC uses content-addressable filenames, you won't encounter issues like
+accidentally overwriting files from one project with another.
+
+A possible issue is that a shared cache will grant all teammates working on a
+given project access to the data from all other projects using that cache. If
+you have sensitive data, you can create different caches for projects involving
+private and public data.
+
+To learn more about setting your cache directory location,
+[see our docs](https://dvc.org/doc/command-reference/cache/dir).
+
+## CML questions
+
+### Q: I use Bitbucket. Will CML work for me?
+
+The first release of CML is compatible with GitHub and GitLab. We've seen
+[many requests for Bitbucket support](https://github.com/iterative/cml/issues/140),
+and we're actively investigating how to add this. Stay tuned.
+
+### [Q: I have on-premise GPUs. Can CML use them to execute pipelines?](https://discordapp.com/channels/485586884165107732/728693131557732403/730070747388706867)
+
+Yep! You can use on-premise compute resources by configuring them as self-hosted
```

Comment:
does it make sense to include links to our examples with self-hosted runners here?

Reply:
We don't have a use case with an on-premise self-hosted runner, only deploying an EC2 or GCP CE instance from the runner, which isn't what this person wants. So I think it does not make sense

Reply:
yep, makes sense.